azurelinuxagent/ga/exthandlers.py

```diff
@@ -381,7 +381,8 @@ def _extension_processing_allowed(self):
 
     def handle_ext_handlers(self, etag=None):
         if not self.ext_handlers.extHandlers:
-            logger.verbose("No extension handler config found")
+            # This is a valid case, some extensions dont have any settings. Just log and exit processing
```

Comment:
not sure I get the comment. this was checking whether there were any extensions to execute... what settings does the comment refer to? not to the settings passed to the extension, does it?

Reply:
Sorry wrong comment, got confused with something here, removed it.